src/gui/FloatingToolbox.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Xournal++
+ *
+ * The Floating Toolbox
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#pragma once
+
+
+#include "GladeGui.h"
+
+class MainWindow;
+
+enum FloatingToolBoxState {
+		recalcSize = 0,
+		configuration,
+		noChange
+};
+
+/************************
+ * FloatingToolbox:
+ *  Uses a GtkOverlay to show a widget containing toolbars.
+ *  The getOverlayPosition callback is used to position the toolbox widget relative to the users input device.
+ *  The ui/main.glade file defines the floatingToolbox as a GtkRevealer widget - attempts to use another widget 
+ *  had issues with leave-notify-event.
+ * 
+ * 
+ * 
+ * 
+ */
+
+class FloatingToolbox 
+{
+public:
+	FloatingToolbox(MainWindow* theMainWindow, GtkOverlay *overlay);
+	virtual ~FloatingToolbox();
+
+public:
+	void show(int x, int y);
```

Comment:
Add documentation here that indicates the coordinate system of x and y.

Reply:
fixed